src/data/AkitaOriginData.js

```diff
@@ -74,14 +77,26 @@ class AkitaOriginData {
 	updatePaymentData(paymentData) {
 		if (paymentData) {
 			const paymentPointer = paymentData.paymentPointer;
-			
+
 			if (paymentPointer) {
 				this.isCurrentlyMonetized = true;
 
 				if (!this.paymentPointerMap[paymentPointer]) {
-					this.paymentPointerMap[paymentPointer] = new AkitaPaymentPointerData(paymentPointer);
+					this.paymentPointerMap[paymentPointer] = new AkitaPaymentPointerData(
+						paymentPointer
+					);
```

Comment:
nit: I think this can just be a single line. `AkitaPaymentPointerData()` only takes 1 param

Reply:
okay. done.